src/constants/tables.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  reservedTableNames: [
```

Comment:
I think it's better to have these as an object, e.g.
```
{
    _USERS: "_users",
    ...
}
```

Reply:
This comment is fixed in the next PR https://github.com/thevahidal/soul/pull/160